# OPTICAL_FLOW_RAD message data

Header header
uint32_t integration_time_us
float integrated_x
float integrated_y
float integrated_xgyro
float integrated_ygyro
float integrated_zgyro
int16_t temperature
uint8 quality
uint32_t time_delta_distance_us
float distance
